.github/scripts/reviewers_add.js

```diff
@@ -110,7 +102,6 @@ module.exports = async ({ github, context }) => {
   }
 
   if (labels.includes("vim-patch")) {
-    reviewers.add("seandewar");
     reviewers.add("zeertzjq");
```

Comment:
TODO: remove

Reply:
?

Reply:
Idk I assumed you didn't want it. I'll leave it then.